### ISSUE TITLE
stay in the same place on the page when content is refreshed.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2310]
**Reported by ashleycox on 2012-05-09 08:50**
When NVDA +f5 is pressed to refresh the content on a page, nvda should return the focus to the place on the page where the user was located before they refreshed the content, if possible.

**Comment by ehollig** [comment]
Closing because of lack of an example demonstrating the issue. Also, NVDA+f5 is somewhat of an emergency command and should not be necessary, as stated by @jcsteh in https://github.com/nvaccess/nvda/issues/2310#issuecomment-155297663.
